scss/_custom.scss

```diff
@@ -379,7 +387,7 @@ $input-height:                  calc(#{$input-height-inner} + #{$input-height-bo
 $input-height-inner-sm:         ($font-size-sm * $input-btn-line-height-sm) + ($input-btn-padding-y-sm * 2) !default;
 $input-height-sm:               calc(#{$input-height-inner-sm} + #{$input-height-border}) !default;
 
-$input-height-inner-lg:         ($font-size-sm * $input-btn-line-height-lg) + ($input-btn-padding-y-lg * 2) !default;
+$input-height-inner-lg:         ($font-size-lg * $input-btn-line-height-lg) + ($input-btn-padding-y-lg * 2) !default;
```

Comment:
there must be a reason why i originally made the a "..size-sm". Helps being able to see the visual execution.

nonetheless - not a big deal. Easy update later, if off.